### ISSUE TITLE
pulling port status on p4

### ISSUE THREAD

**Issue by gcooper** [p4lang/p4factory#85]
Is there a way to get the status of a port ( whether its up or down ) in p4?

**Comment by antoninbas** [comment]
No, there is not yet a way to express this in P4. Your application has to poll port status and update table entries accordingly.

**Comment by gcooper** [comment]
by application, are you referring to an SDN application? ( like a controller )

**Comment by antoninbas** [comment]
Yes, that's what I meant.
